services/ui-src/src/types/index.ts

```diff
@@ -113,12 +113,19 @@ export interface ReportContextShape extends ReportContextMethods {
 
 // FORM & FIELD STRUCTURE
 
+export declare type PageEntityType =
+  | "plan"
+  | "bssEntity"
+  | "qualityMeasures"
+  | "someOtherEntity";
```

Comment:
Did we have specific values for this? Is it just the first 3 for now?

Reply:
Yep it's actually just the first two for now. We're actually going to be changing the names of the fields too, so the field name for plans will not be `aap-1`, it will be `plans`, likewise with `arp-1` → `bssEntities`.